### ISSUE TITLE
Redis cluster - no replicas

### ISSUE THREAD

**Issue by davejhahn** [bitnami/charts#2670]
redis-cluster chart

Relatively new to Redis, so maybe I'm not understanding this entirely.  But, when I create a cluster, say with the default 6 nodes, the default replica for each node is 1.  Shouldn't I expecting to see one replica / slave pod for each node?  e.g. if I create a 3 node cluster, with one replica, shouldn't I see 3 nodes and 3 slaves?  If I change cluster.replicas=2 I still don't see anything.  Changing cluster.nodes is visible to the appropriate count.  

I thought maybe there was a difference with the production values, but it appears to be the same for both.   

I'm sure I'm not understanding it correctly, but I'm not sure what based on what I was reading.

**Comment by davejhahn** [comment]
I think I figured it out.....first, my confusion is in pod names.  Slaves and Master pods are both called the same thing (e.g. redis-redis-cluster-n).   Once I used the cluster-cli to get more information about the nodes I found the slave instances.

Also, the node count is not the master count, it is the total slaves + masters required.

So the formula is something like:

Given # masters desired (m) and # replicas desired per master (r)
(s) slave count = m * r
(n) total node count = s + m
